stellar-dotnet-sdk/TransactionBase.cs

```diff
@@ -104,6 +104,52 @@ public void Sign(byte[] preimage)
             Signatures.Add(decoratedSignature);
         }
 
+        /// <summary>
+        /// Add a signature to the transaction. Useful when a party wants to pre-sign
+        /// a transaction but doesn't want to give access to their secret keys.
+        /// This will also verify whether the signature is valid.
+        /// </summary>
+        /// <param name="publicKey">The public key of the signer</param>
+        /// <param name="signature">The base64 value of the signature XDR</param>
+        public void Sign(string publicKey, string signature)
```

Comment:
Unsure if `IAccountId` is better practice for just a public key. It appears in `Asset.cs` we often take a string, so I did the same here.

Happy to change this if there is a better practice to follow.